orbit/src/main/java/com/babylon/orbit/BaseOrbitContainer.kt

```diff
@@ -107,6 +110,7 @@ class BaseOrbitContainer<STATE : Any, SIDE_EFFECT : Any>(
 
     override fun disposeOrbit() {
         disposables.clear()
+        executor.shutdown()
```

Comment:
if the executor is never used before orbit is disposed this will create it just to shut it down, maybe this scenario is unrealistic, but thought it worth highlighting for comment

Reply:
Good point, will solve this in another way then.